src/main/java/com/learn/hashmap/Customer.java

```diff
@@ -0,0 +1,14 @@
+package com.learn.hashmap;
+
+public class Customer {
+    private final String id;
+
+    public Customer(String cusId) {
```

Comment:
Avoid using abbreviations like `custId` when defining variable , these programs are for humans to read , java is very generous when it comes to variable name lengths. Take the advantage of it . A better name would be `customerId`.

Reply:
Changed variable name, cusId to customerId